### PR TITLE
fix(datetime): standardise order of controls in Storybook

### PR DESCRIPTION
**Pull request by nathalielindqvist**
**Describe pull-request**  
Standardised order of controls in Storybook for datetime component. Also updated descriptions, refactored noMinWidth control and remover unnecessary controls.

**Solving issue**  
Fixes: [AB#3431](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3431)

**How to test**  
1. Go to Storybook link below
2. Check in Datetime -> All sub stories
3. Inspect order of controls
4. Read control descriptions

**Suggested test steps**
- [ ] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [ ] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

### CODE REVIEW

File: tegel/src/components/datetime/datetime.stories.tsx [hunk 1]
```diff
@@ -31,112 +31,109 @@ export default {
         defaultValue: { summary: 'Inherit from parent' },
       },
     },
-    type: {
-      name: 'Type',
-      description: 'Set the field to display date, time or both',
+    state: {
+      name: 'State',
+      description: 'Switches between success or error state.',
```
Comment (EmelieLitwin) [inline tegel/src/components/datetime/datetime.stories.tsx:36]: Should it be 'Switches between success AND error state.'?

**Comment by EmelieLitwin** [comment]
This is what it looks if we allow it to shrink: 
image
Maybe we shouldn't allow for that?

**Comment by nathalielindqvist** [comment]
> This is what it looks if we allow it to shrink: image Maybe we shouldn't allow for that?

Nice catch! Got some IRL feedback from others in the team and it might still be a nice feature for the user to have in case they want a more flexible component. But I updated the width of the demo wrapper, which is what kicks in when enabling the noMinWidth prop; so it doesn't look so ugly when it shrinks in Storybook 😄

**Review by mJarsater — APPROVED** [review]
🚀

**Pull request merged**